vita3k/app/src/app_init.cpp

```diff
@@ -157,6 +158,7 @@ void init_paths(Root &root_paths) {
         root_paths.set_config_path(base_path);
         root_paths.set_shared_path(base_path);
         root_paths.set_cache_path(base_path / "cache" / "");
+        root_paths.set_patch_path(base_path / "patch" / "");
 
 #if defined(__linux__) && !defined(__ANDROID__) && !defined(__APPLE__)
         // XDG Data Dirs.
```

Comment:
There should also be a set_patch_path for linux (preferably right before the endif) that looks something like:
```cpp
root_paths.set_patch_path(shared_path / "patch" / "");
```
Linux's base_path isnt always writable

Reply:
Good call 👍